hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMMetadataManager.java

```diff
@@ -327,4 +327,11 @@ String getMultipartKey(String volume, String bucket, String key, String
    */
   <KEY, VALUE> long countEstimatedRowsInTable(Table<KEY, VALUE> table)
       throws IOException;
+
+  /**
+   * Return the existing upload keys which includes volumeName, bucketName,
+   * keyName and uploadId.
```

Comment:
Upload is not passed. Need to update javadoc.

Reply:
Thanks.